src/psyclone/psyir/symbols/datasymbol.py

```diff
@@ -70,21 +69,12 @@ class DataSymbol(Symbol):
     '''
     def __init__(self, name, datatype, visibility=Symbol.DEFAULT_VISIBILITY,
                  constant_value=None, interface=None):
-        super(DataSymbol, self).__init__(name, visibility)
+        super(DataSymbol, self).__init__(name, visibility, interface)
 
         self._datatype = None
         self.datatype = datatype
-
-        # The following attributes have setter methods (with error checking)
-        self._interface = None
+        # The following attribute has a setter method (with error checking)
```

Comment:
This is also true for datatype, so I would make this comment plural and move it to the top

Reply:
Well spotted.